src/components/common/Tabs/TabContent.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { TabItemType } from 'components/OrderBuySell'
+
+type Props = {
+  tabItems: TabItemType[]
+  activeTab: number
+}
+
+const TabContent: React.FC<Props> = (props) => {
+  const { tabItems, activeTab } = props
+  return (
+    <div>
+      {tabItems.map((tab) => {
+        return tab.id === activeTab ? tab.content : ''
```

Comment:
This would render `''` for non-active tabs.
Better do
```suggestion
        return tab.id === activeTab ? tab.content : null
```

to not render anything